distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -221,6 +222,27 @@ Complete the following procedure now that several queries have been executed.
 Based on the browser's configuration, the .xls file will be downloaded or automatically displayed in Excel.
 ====
 
+=== Configuration Export/Import
+==== Export Configuration Settings
+
+1) Launch DDF.
```

Comment:
We should use `.` instead of numbering the list manually. See http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#lists.